Listenbrainz/UI/Screens/ContentView.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  ContentView.swift
+//  Listenbrainz
+//
+//  Created by Gaurav Bhardwaj on 16/04/23.
+//
+
+import SwiftUI
+import UIKit
+
+struct ContentView: View {
+    
+    init() {
+        UITabBar.appearance().unselectedItemTintColor = UIColor.systemBlue
```

Comment:
Most probably, we'll need to fix the icons and not change their colours when pressed.